src/lzf.h

```diff
@@ -73,9 +73,7 @@
  * and lzf_c.c.
  *
  */
-size_t
-lzf_compress (const void *const in_data,  size_t in_len,
-              void             *out_data, size_t out_len);
+size_t lzf_compress(const void *const in_data, size_t in_len, void *out_data, size_t out_len);
```

Comment:
The .clang-format-ignore file I added in the other PR didn't work?

Reply:
hmm - don't know why. let me revert all changes and reapply the formatting.

Reply:
I think my clang-format is too old. It is 14.0. I have been getting all kinds of errors when trying to update it on my machines. Will need to find a different machine and try again.

Reply:
This can be a problem for users I suppose. An old version reformats lots of files and it gets committed. We could add the comment /* clang-formt off */ in all these files instead if it's safer.

Reply:
"clang-format off" is an option but for "strong consistency" I think we should introduce github actions that use  the same clang-format version to format any PR before it gets merged.

Reply:
You want to autoformat on merge? A drawback is how it wraps long comments if they're indented as in this comment: https://github.com/valkey-io/valkey/pull/323#pullrequestreview-2046015165. For that (and some other situations) it'd be better to have clang-format fail CI and let the author fix it.

Reply:
Here's a github action for *checking*: https://github.com/jidicula/clang-format-action